### DISCUSSION TITLE
Welcome to datav discussions

### DISCUSSION THREAD

**Discussion by sunface** [deepops-ai/deepops#236] [General]
Please feel free to leave a message here :)

**Comment by t00350320** [comment]
hi, sunface
1、How does your platform work with otlp 
2、Will it transfers metrics/traces/logs to special format data and handle long-term storage
3、differences compared with grafana UI or OpenObserve ,etc

**Reply by sunface** [reply]
Hi, @t00350320 .

It seems  you are coming from China, if so, you can contact me through **wechat**: `isunface` .

I'm very happy you are interesting in Datav. Looking forward to your messages :)
